pypowsybl/network.py

```diff
@@ -4124,6 +4124,228 @@ def remove_extensions(self, extension_name: str, ids: _Union[str, _List[str]]) -
             ids = [ids]
         _pp.remove_extensions(self._handle, extension_name, ids)
 
+    def create_load_bay(self, busbar_section_id: str, load_position_order: int, df: _DataFrame,
```

Comment:
To be homogeneous with other network modifications, I would move those methods outside the `Network` class.

Alternatively, we could imagine giving the possibility of creating bays in the existing creation methods (`create_loads` etc).

@annetill @geofjamg : any opinion about this last proposition ?